README.md

```diff
@@ -0,0 +1,95 @@
+# FarmLink
+Our capstone project is a web application that helps FarmLink, a non-profit organization that moves food surplus from farms around the country to food banks in need, automate the process of matching a food bank with a farm.
+# How to get application running!
+In this README file we will briefly explain how to get this application up and running on your computer. Feel free to reach out to the contributors to the repo for any further assistance.
+## Step #1: Clone the Repository
+Navigate to the directory where you would like to clone the repository. Then navigate to the main page of the repository, press the green code button, and then copy the link under "Clone with HTTPS".
+
+Now enter the following command into terminal:
+
+    git clone https://hostname/YOUR_USERNAME/YOUR-REPOSITORY
+At this point all the folders within the repository should have successfully downloaded locally.
```

Comment:
This feels _too_ in depth. You could just link to another one of the myriad of git docs that explain how to clone a directory.

Reply:
Agreed, I'd just link to GitHub's documentation for this.